src/pages/home/report/ReportActionsView.js

```diff
@@ -36,6 +36,7 @@ import {contextMenuRef} from './ContextMenu/ReportActionContextMenu';
 import PopoverReportActionContextMenu from './ContextMenu/PopoverReportActionContextMenu';
 import variables from '../../../styles/variables';
 import MarkerBadge from './MarkerBadge';
+import {markEnd, markStart, withRenderTrace} from '../../../libs/Performance';
```

Comment:
Same comment about maybe using a default export here or combining into a `Performance` module. I think it would make things a bit easier to differentiate what is debug metrics vs. production timing.